WcaOnRails/config/locales/de.yml

```diff
@@ -3498,8 +3689,8 @@ de:
           #original_hash: daf1cb1
           '2': >-
             Abhängig von der Größe des Wettbewerbs und dem persönlichen
-            Terminkalender des WCA-Delegierten, kann dies nur einen Tage oder
-            bis zu einer Woche in Anspruch nehmen. Bitte übe dich in Geduld.
+            Terminkalender des WCA-Delegierten, kann dies nur einen Tag oder bis
```

Comment:
"von nur einem Tag bis zu..."

Reply:
actually only wanted to fix the typo here :D do you want to rewrite the sentence completely, to use sth. like "kann dies eine Zeitspanne von nur einem Tag bis zu einer Woche in Anspruch nehmen"?

Reply:
yeah, I like that!